_data/simple-icons.json

```
@@ -8258,6 +8258,11 @@
             "hex": "2FCEA0",
             "source": "https://www.skyliner.io/help"
         },
+        {
+            "title": "Skynet",
+            "hex": "00C65C",
+            "source": "https://siasky.net/"
```

This should probably also be updated to `https://support.siasky.net/key-concepts/skynet-brand-guidelines`, and we should definitely add that as a guidelines. 

However, it seems you used the SVG from the header of `https://siasky.net/` as source, is that correct? We will review the SVG after you answered this question.

```suggestion
            "source": "https://support.siasky.net/key-concepts/skynet-brand-guidelines",
            "guidelines": "https://support.siasky.net/key-concepts/skynet-brand-guidelines"
```

---

Hi, in this case I actually took the SVG from the footer of the homepage (https://siasky.net/). Does that change anything here?

---

Thanks for clarifying. Following my review of the SVG, can you update this as per my request? It's generally a better idea to use the actual brand guidelines as a source.